flake.nix

```diff
@@ -7,17 +9,23 @@
 
     flake-parts.url = "github:hercules-ci/flake-parts";
     nixos-flake.url = "github:srid/nixos-flake";
+
+    # see https://github.com/nix-systems/default/blob/main/default.nix
+    systems.url = "github:nix-systems/default";
   };
 
-  outputs = inputs@{ self, ... }:
+  outputs = inputs:
     inputs.flake-parts.lib.mkFlake { inherit inputs; } {
-      systems = [ "x86_64-linux" "aarch64-darwin" "x86_64-darwin" ];
+      systems = import inputs.systems;
       imports = [
         inputs.nixos-flake.flakeModule
+        # Edit the contenst of the ./home directory to install packages and modify dotfile configuration in your
+        # $HOME.
+        #
+        # https://nix-community.github.io/home-manager/index.html#sec-usage-configuration
+        ./home
```

Comment:
To address [this comment](https://github.com/juspay/nix-dev-home/pull/7/files#r1320630741) we would remove this line.

Reply:
https://github.com/juspay/nix-dev-home/pull/7/commits/968444174202d146ab598b0d5c215dcae5b810c2